samples/net/sockets/dumb_http_server_mt/src/main.c

```diff
@@ -165,7 +165,7 @@ static void client_conn_handler(void *ptr1, void *ptr2, void *ptr3)
 		 * We are assuming here that the full HTTP request is received
 		 * in one TCP segment which in real life might not.
 		 */
-		if (strstr(buf, "\r\n\r\n")) {
+		if (strstr(buf, "\r\n\r\n\0")) {
```

Comment:
the original cid seems like a false positive to me, afaik the '\0' is implicitly added in constant strings.

Reply:
> implicitly added in constant strings.

missed that. I checked in manpages of strstr. It expects '\0' but it is not compared. I will close it as false positive.